packages/components/calendar/src/calendar.stories.ts

```diff
@@ -89,14 +162,41 @@ export const Readonly: Story = {
 
 export const Selected: Story = {
   args: {
-    selected: new Date(2024, 8, 15)
+    month: new Date(1755640800000),
+    selected: new Date(1755640800000),
+    showToday: true
+  }
+};
+
+export const IndicatorDates: Story = {
+  args: {
+    indicatorDates: [
+      { date: new Date(), color: 'red', label: indicatorLabels.red.label },
+      { date: new Date('2025-09-05'), color: 'blue' as IndicatorColor, label: indicatorLabels.blue.label },
+      { date: new Date('2025-09-24'), label: indicatorLabels.default.label },
+      { date: new Date('2025-09-09'), color: 'green' as IndicatorColor, label: indicatorLabels.green.label },
+      { date: new Date('2025-09-11'), color: 'grey' as IndicatorColor, label: indicatorLabels.grey.label },
+      { date: new Date('2025-09-12'), color: 'yellow' as IndicatorColor, label: indicatorLabels.yellow.label },
+      { date: new Date('2025-09-18'), color: 'red', label: indicatorLabels.red.label }
+    ],
+    month: new Date('2025-09-01'),
+    showToday: true
+  }
+};
+
+export const DisabledDates: Story = {
```

Comment:
In this story, when I use only arrow right key to move to the next available days, when it moves to the next month it selects (even when I'm not selecting anything) the first day of next month and there when I use arrow right key again it jumps to the next month (which is disabled, so should not be available).

https://github.com/user-attachments/assets/330ff353-2e2f-432d-b00b-5b6277477a26